packages/react-api/src/api/SQL.js

```diff
@@ -129,7 +129,9 @@ function createRequest({
     connection,
     parameters: urlParamsForPost
   });
-  return postRequest(postUrl, payload, requestOpts);
+  return postRequest(postUrl, payload, requestOpts, {
```

Comment:
One question, I don't see the check for V3 here, so I guess we're always sending the bearer token? Even for V1 and V2?

Reply:
Oh, thanks so much! I didn't take that into account 🙇🏼